cmd/util.go

```diff
@@ -213,10 +213,12 @@ func writeManifestFile(decoded []byte, filename string, toAppend bool) error {
 		log.Error(err)
 		return err
 	}
+	defer f.Close()
 	if toAppend {
 		f.WriteString("---\n")
 	}
-	defer f.Close()
+	// Remove newline from the decoded slice if the slice starts with a newline
+	decoded = []byte(strings.TrimPrefix(string(decoded), "\n"))
```

Comment:
Optionally, if you want to get around converting twice
```
if decoded[0] == '\n' {
     decoded = byte[1:]
}
```

Reply:
I thought TrimPrefix was a cleaner approach and very little performance difference. @klautcomputing can you weigh in?

Reply:
tbh: I don't care too much either way.